src/StateService/README.md

```diff
@@ -0,0 +1,60 @@
+# StateService
+
+## RPC API
+
+### GetStateRoot
+#### Params
+|Name|Type|Summary|Required|
+|-|-|-|-|
+|Index|uint|index|true|
+#### Result
+StateRoot Object
+|Name|Type|Summary|
+|-|-|-|
+|version|number|version|
+|index|number|index|
+|roothash|string|version|
+|witness|Object|witness from validators|
+
+### GetProof
+#### Params
+|Name|Type|Summary|Required|
+|-|-|-|-|
+|RootHash|UInt256|state root|true|
+|ScriptHash|UInt160|contract script hash|true|
+|Key|base64 string|key|true|
+#### Result
+Proof in base64 string
+
+### VerifyProof
+#### Params
+|Name|Type|Summary|
+|-|-|-|
+|RootHash|UInt256|state root|true|
+|Proof|base64 string|proof|true|
+#### Result
+Value in base64 string
+
+### GetStateheight
+#### Result
+|Name|Type|Summary|
+|-|-|-|
+|localrootindex|number|root hash index calculated locally|
+|validatedrootindex|number|root hash index verified by validators|
+
+### GetState
```

Comment:
docs need to be updated to match implementation

Reply:
Will do.

Reply:
Updated.